js/deleteTask.js

```diff
@@ -0,0 +1,11 @@
+//import { taskList } from "/js/addToTask.js";
+import { CLICK_EVENT, DELETE, UL_LIST } from "/js/constants.js";
+
+export default function removeTodos(element, taskList) {
+  let index = taskList.findIndex((item) => {
+    return item.id == element.id;
+  });
+  taskList[index].trash = true;
+  element.parentNode.parentNode.removeChild(element.parentNode);
```

Comment:
this is too error prone (accessing the grand parent of an element through `parentNode`). did you know there is a `.remove()` method available on the element which you can use to remove the element from the DOM?

// see https://developer.mozilla.org/en-US/docs/Web/API/Element/remove